src/pages/graphql/payment-services-extension/mutations/set-cart-inactive.md

```diff
@@ -0,0 +1,63 @@
+---
+title: setCartAsInactive mutation
+description: This mutation marks a cart as inactive in Payment Services, helping prevent multiple active carts for logged-in customers during specific payment flows.
+---
+
+# setCartAsInactive mutation
+
+<InlineAlert variant="info" slots="text" />
+
+This mutation is available only if you have installed [Payment Services for Adobe Commerce](https://commercemarketplace.adobe.com/magento-payment-services.html) 2.12.0 or higher.
+
+The `setCartAsInactive` mutation sets a specific `cartId` as inactive. It is recommended to be used in case of an error, or cancel callback of an Apple Pay specific payment flow, to avoid having multiple active carts for logged-in customers created with the [`AddProductsToNewCart`](add-products-new-cart.md) mutation.
+
+## Syntax
+
+```graphql
+mutation {
+    setCartAsInactive(
+        cartId: "gVFVNBSJPt7L6uCQis4Q1ryj6DQtcwM2"
+    ) {
+           success
+           error 
+        }
+    }
+```
+
+## Example usage
+
+These examples show when the `setCartAsInactive` mutation return a success, or errors, when adding a product to a new cart for logged-in customers.
```

Comment:
You're confused here. The response of `addProductsToNewCart` is irrelevant. The `setCartAsInactive` mutation can be run at any time, perhaps when something goes wrong at checkout, or maybe when it's obvious the shopper has abandoned the cart. 

Just show the mutation and the success response. A `success` value of `false` isn't tricky, so showing it doesn't add much. If you do keep the false response, I have to say the shown error looks wrong. Considering the "=", I would expect the value there.